tasks/Task.hpp

```diff
@@ -83,6 +83,7 @@ tasks/Task.cpp, and will be put in the usbl_seatrac namespace.
          * it again. Finally, FatalError cannot be recovered.
          */
         void updateHook();
+        void processIO();
```

Comment:
I think you should put that as `protected`...or even `private`

Reply:
The 'Task::processIO' is from iodrivers_base, I needed override it even I don't use it. I guess that I can move it to private, but I am not sure.

Reply:
Tbh,  the visibility on task context classes is really unimportant. They are very rarely subclassed.